rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -261,7 +267,7 @@ public void enableRemasc() {
 
     public long peerDiscoveryMessageTimeOut() {
         return configFromFiles.hasPath("peer.discovery.msg.timeout") ?
```

Comment:
Please take this chance to move the default to `reference.conf` and remove this `hasPath` check.

Reply:
I will address this on another PR